smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/swiftmodules/ClientRuntimeTypes.kt

```diff
@@ -50,6 +50,8 @@ object ClientRuntimeTypes {
         val MutableInput = runtimeSymbol("MutableInput", SwiftDeclaration.PROTOCOL)
         val Handler = runtimeSymbol("Handler", SwiftDeclaration.PROTOCOL)
         val OrchestratorBuilder = runtimeSymbol("OrchestratorBuilder", SwiftDeclaration.CLASS)
+        val OrchestratorTelemetry = runtimeSymbol("OrchestratorTelemetry", SwiftDeclaration.CLASS)
+        val OrchestratorMetricsAttributesKeys = runtimeSymbol("OrchestratorMetricsAttributesKeys", SwiftDeclaration.ENUM)
```

Comment:
Add `Orchestrator` related symbols for codegen per operation, particular for the `rpc.service` and `rpc.method` attributes.